module/behaviour/skills/HeadBehaviourSoccer/src/HeadBehaviourSoccer.cpp

```diff
@@ -59,9 +59,37 @@ namespace module::behaviour::skills {
     using utility::math::coordinates::sphericalToCartesian;
     using utility::math::geometry::Quad;
     using utility::motion::kinematics::calculateCameraLookJoints;
-    using utility::nusight::graph;
     using utility::support::Expression;
 
+
+    /// @brief Converts from camera space direction to IMU space direction
+    Eigen::Vector2d getIMUSpaceDirection(const KinematicsModel& kinematicsModel,
```

Comment:
Can we put in anon namespace?

Reply:
This got moved into the class since this review.